.github/workflows/auto-job-v2.yaml

```diff
@@ -0,0 +1,29 @@
+name: auto-job-v2
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  add-v2-label:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Add v2 label if target branch is feature/v2
+        if: github.event.pull_request.base.ref == 'feature/v2'
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: 'v2'
+
+   add-e2e-comment:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Add comment if target branch is feature/e2e
+        if: github.event.pull_request.base.ref == 'feature/e2e'
```

Comment:
"feature/e2e" branch? But this PR is created for `feature/v2` branch.

Reply:
AI mistake